nixos/modules/tasks/network-interfaces-scripted.nix

```diff
@@ -251,6 +251,11 @@ let
                 ${i}
               '')}" > /run/${n}.interfaces
 
+              # Enslave previously attached interfaces
+              [ -f /run/${n}.slaves ] && for ifname in `cat /run/${n}.slaves`; do
```

Comment:
As I said in a comment on the whole pull request, I don't like the idea here. If a service (container) is depending on the bridge, the dependency should be defined so that systemd can stop and start the dependencies accordingly.

The code above this change "Save list of enslaved interfaces" is saving this services state _while it is running_.

What you are doing is saving state _while the service is not running_. Which I believe is really stuff that either should be 'saved' by the whole nix packaging/configuration thing or not need saving at all.